src/photon.py

```diff
@@ -0,0 +1,15 @@
+import astropy.units as u
+import numpy as np
+
+from einsteinpy import constant
+
+''' any ideas for any existing base class this can inherit ?'''
+
+class photon:
```

Comment:
Do we really need this class?

Reply:
I think a separate class would be more practical than defining the radiative transfer params in each of spacetime types so that any arbitrary spacetime would use this class to contain these params ?